arbnode/resource_management_test.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2023, Offchain Labs, Inc.
+// For license information, see https://github.com/nitro/blob/master/LICENSE
+
+package arbnode
+
+import (
+	"fmt"
+	"os"
+	"testing"
+)
+
+func updateFakeCgroupv1Files(t *testing.T, c *cgroupsV1MemoryLimitChecker, limit, usage, inactive int) {
```

Comment:
Return error from here instead, and check it at call site.

https://google.github.io/styleguide/go/decisions#assertion-libraries: "For more domain-specific comparison helpers, prefer returning a value or an error that can be used in the test’s failure message instead of passing *testing.T and calling its error reporting methods".

Reply:
Removed uses of `Require` and `Fail`